src/renderers/WebGLRenderer.js

```diff
@@ -2080,47 +2080,90 @@ function WebGLRenderer( parameters = {} ) {
 
 	};
 
-	this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level = 0 ) {
+	this.copyTextureToTexture = function ( srcTexture, dstTexture, dstPosition = undefined, srcBox = undefined, dstLevel = 0, srcLevel = 0 ) {
+
+		if ( ! srcTexture.isTexture ) {
+
+			console.warn( 'THREE.WebGLRenderer: .copyTextureToTexture() now expects (srcTexture, dstTexture, dstPosition, srcBox, dstLevel, srcLevel).' );
+			return this.copyTextureToTexture( arguments[ 1 ], arguments[ 2 ], arguments[ 0 ], undefined, arguments[ 3 ] );
+
+		}
+
+		const image = srcTexture.mipmaps[ srcLevel ] || ( srcLevel == 0 ? srcTexture.image : null );
+		if ( ! image ) {
+
+			console.error( 'mipmap level ', srcLevel, ' is not available in srcTexture' );
+			return;
+
+		}
+
+		let skipPixels = srcBox ? srcBox.min.x : 0;
+		let skipRows = srcBox ? srcBox.min.y : 0;
+		let x = dstPosition ? dstPosition.x : 0;
+		let y = dstPosition ? dstPosition.y : 0;
+
+		// ensure skipPixels, skipRows, x and y >= 0
+		if ( skipPixels < 0 || x < 0 ) {
```

Comment:
I think both of those conditions aren't needed and should work the same way `srcBox` is clamped to `0` when negative.
They add a lot of code for something the user should clearly not do. We can add a little disclaimer in the doc